lib/index.js

```
@@ -12,7 +12,9 @@ const Wreck = require('wreck');
 const internals = {
     defaults: {
         threshold: 20,
+        errorThreshold: 0,
         schema: 'good-http',
+        ignoreRequestErrors: false,
```

You'd set `errorThreshold` to `Infinity` to ignore all errors I would think. Default it to `0` so it works like it does now, `Infinity` means ignore all, and a number means allow that many like per log attempt.

Sound reasonable to you?